test/support/setupTestFramework.js

```diff
@@ -1,3 +1,25 @@
+require('babel-register')({
+  presets: ['es2015', 'es2016', 'react-native'],
+  ignore: /node_modules\/(?!react-native)/,
+})
+require('react-native-mock/mock')
 
-window.shallowRender = require.requireActual('./shallowRender')
-jest.setMock('react-native', require.requireActual('../../__mocks__/react-native'))
+const jsdom = require('jsdom')
```

Comment:
Enzyme needs a "dom" context to render into, so we create a small jsdom and expose it as globals.